### PR TITLE
Fix non-normative example of HTTP response in the Presentation Remote Flow

### LINKED ISSUE #355: response_code in the Presentation Flow
In the last step of the Presentation flow the RP provides the Wallet Instance with a `redirect_uri` parameter within the HTTP response. The   non-normative example is the following one.

```
HTTP/1.1 200 OK
Content-Type: application/json

{
  "redirect_uri": "https://relying-party.example.org/cb#response_code=091535f699ea575c7937fa5f0f454aee"
}
```

The response_code MUST be given as a query parameter as follows.

```
HTTP/1.1 200 OK
Content-Type: application/json

{
  "redirect_uri": "https://relying-party.example.org/cb?response_code=091535f699ea575c7937fa5f0f454aee"
}
```

### PR DESCRIPTION
**Pull request by fmarino-ipzs**
## Content
Closes #355 
## Review

- [ ] Ensure your files are written following RST specs (*not MD!*)
- [ ] Italian version
- [ ] English version
- [ ] Example files 
- [ ] Ask for review

### CODE REVIEW

**Review by peppelinux — APPROVED** [review]

File: docs/en/remote-flow.rst [hunk 1]
```diff
@@ -532,10 +532,10 @@ The following is a non-normative example of the response from the Relying Party
 .. code-block:: http
 
   HTTP/1.1 200 OK
-  Content-Type: application/json;charset=UTF-8
+  Content-Type: application/json
 
   {
-    "redirect_uri": "https://relying-party.example.org/cb#response_code=091535f699ea575c7937fa5f0f454aee"
+    "redirect_uri": "https://relying-party.example.org/cb?response_code=091535f699ea575c7937fa5f0f454aee"
```
Comment (peppelinux) [inline docs/en/remote-flow.rst:538]: the reasn why it was a fragment is to allow the registration of the redirect_uris in the metadata and using fragments to allow randomization of the endpoints

Comment (grausof) [reply docs/en/remote-flow.rst:538]: Please, see also https://github.com/node-fetch/node-fetch/issues/211

**Review by grausof — APPROVED** [review]

File: docs/en/remote-flow.rst (hunk 1 above)
Comment (peppelinux) [reply docs/en/remote-flow.rst:538]: openid4vp non-normative example uses the fragment
https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-6.2-22

File: docs/en/remote-flow.rst [hunk 2]
```diff
@@ -532,10 +532,10 @@ The following is a non-normative example of the response from the Relying Party
...
   }
 
 The ``redirect_uri`` value MUST be used with an HTTP method GET by either the Wallet Instance or the user-agent to redirect the User to the Relying Party in order to complete the process. The specific entity that performs this action depends on whether the flow is Same device or Cross device.
```
Comment (fmarino-ipzs) [inline docs/en/remote-flow.rst]:
```suggestion
The ``redirect_uri`` value MUST be used with an HTTP method GET by either the Wallet Instance or the user-agent to redirect the User to the Relying Party in order to complete the process. The value can be added as a path component, as a fragment or as a parameter to the URL according to Section 6.2 of `OID4VP`_. The specific entity that performs this action depends on whether the flow is Same device or Cross device.
```

**Review by peppelinux — APPROVED** [review]

**Pull request merged**